server/controllers/request.js

```diff
@@ -0,0 +1,104 @@
+const Request = require("../models/Request");
+const User = require("../models/User");
+const asyncHandler = require("express-async-handler");
+const mongoose = require("mongoose");
```

Comment:
Prefer `require`ing these packages at the top of your file before your own modules (`Request` and `User`). It's just good practice.

Reply:
Thanks, I'll fix that in the next commit